developers/bindings/thing-xml.md

```diff
@@ -994,3 +994,37 @@ The contents of this list are dynamic, and it may be extended from time to time.
 If you are an addon developer and you think there is something missing from the list please open an [Issue](https://github.com/openhab/openhab-core/issues) or [Pull Request](https://github.com/openhab/openhab-core/pulls) on GitHub.
 
 1. For further reading please see the [Description of the Semantic Model]({{base}}/tutorial/model.html), the [Developer Guidelines on Semantic Tags](semantic-tags.md) and the [Thing-Type and Channel-Type validation schema](https://www.openhab.org/schemas/thing-description-1.0.0.xsd).
+
+## Tagging Conventions for Commonly Confused Use Cases
+
+The following are some use cases that commonly lead to confusion.
+The purpose of this list is to provide the convention for tagging such cases:
+
+1. In the case of `Point` and `Property` tags it may help to consider the `Point` as the VERB and the `Property` as the OBJECT in a sentence.
+So a channel may make a 'Measurement' of a 'Temperature', or show the 'Status' of an operating 'Mode' etc.
+
+1. Do not confuse `Property` tags with Units of Measure.
+A `Property` tag is a WORD (see above) that describes the nature an the action being taken by its respective `Point`.
+So for example `Speed` need not be taken precisely to mean `m/sec` .. but can in general cover operations that "do much stuff in little time".
+
+1. For turning a piece of equipment on or off:
+`Switch.Light` should be used if the equipment is a light, otherwise `Switch.Power` should be used.
+
+1. For switching the operating mode of a piece of equipment (e.g. Auto/Manual, Day/Night, Disable/Enable, etc.):
+`Switch.Mode` should be used.

```

Comment:
Reading this, I'm not sure whether I should use Switch.Mode when there could be more than two modes, e.g. heat, cool, dry, fan, auto. Add some further clarifications maybe?

Reply:
> use Switch.Mode when there could be more than two modes, e.g. heat, cool, dry, fan, auto.

I would say `Control.Mode` in that case.

> Add some further clarifications maybe?

Yes I will do it.

Reply:
So can this apply even when the linked item type is not `Switch`?

Reply:
Yes why not? I would say `Switch.Mode` (for binary `Switch` Items) and `Control.Mode` (for non-binary `String` Items).